.eslintrc.js

```diff
@@ -3,14 +3,24 @@ module.exports = {
     node: true,
     commonjs: true,
     jest: true,
+
   },
+  plugins: [
+    'cypress'
+  ],
   extends: [
-    'airbnb-base'
+    'airbnb-base',
+    'plugin:cypress/recommended'
   ],
   parserOptions: {
     ecmaVersion: 9,
     sourceType: 'module',
   },
   rules: {
```

Comment:
please add this rule `"class-methods-use-this": 0,` then eslint will not bark on absence of this in getters and methods

Reply:
done